ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTests.cs

```diff
@@ -1,6 +1,5 @@
 ﻿using LeetCodeTraining.TwoSum1;
-
-namespace LeetCodeTraining.Tests
+namespace TwoSumTests
```

Comment:
fix namespace, it should be as "LeetCodeTraining.Tests.[FileName]"

Reply:
fixed errors in namespace